extensions/wikia/AdEngine/js/video/uapVideo.js

```diff
@@ -70,11 +87,13 @@ define('ext.wikia.adEngine.video.uapVideo', [
 			});
 
 			video.addEventListener(win.google.ima.AdEvent.Type.ALL_ADS_COMPLETED, function () {
+				console.log("**** cALLING RELOAD");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)